lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -243,7 +239,7 @@ def _get_tracing_endpoint(tracing_endpoint_getter, self, charm):
         tracing_endpoint = tracing_endpoint_getter(self)
 
     if tracing_endpoint is None:
-        logger.warning(
+        logger.debug(
             f"{charm}.{getattr(tracing_endpoint_getter, '__qualname__', str(tracing_endpoint_getter))} "
             f"returned None; continuing with tracing DISABLED."
```

Comment:
This sounds like either normal operation and should log, or misuse of the lib and should raise an exception.

Reply:
It is a normal operation in case tracing endpoint is not defined, for instance because tracing relation isn't set yet. I feel though that if the charm has charm_tracing support it might be something the user might want to know. Again, though, I think we could have a better message.
```suggestion
        logger.debug(
            "Charm tracing is disabled. Tracing endpoint is not defined - tracing is not available or relation is not set."
```

Reply:
+1